pkg/bpmn_engine/engine_jobs_activated.go

```diff
@@ -18,16 +19,32 @@ type activatedJob struct {
 	processDefinitionKey     int64
 	elementId                string
 	createdAt                time.Time
+	scope                    variable_scope.VarScope
```

Comment:
As I wrote in the other comment, an activated Job should not be able to alter its variables when in-flight ... but only when completing or failing. 
Also, a job only should contain one variable Context, not two.
Makes sense?

Reply:
Good idea, I'll remove one variable context.